src/lib/isFieldRequested.ts

```diff
@@ -0,0 +1,46 @@
+import { GraphQLResolveInfo } from "graphql"
+import {
+  parseResolveInfo,
+  ResolveTree,
+  simplifyParsedResolveInfoFragmentWithType,
+} from "graphql-parse-resolve-info"
+
+export const isFieldRequested = (
+  field: string,
+  resolveInfo: GraphQLResolveInfo
+) => {
+  let parsedResolveInfoFragment
```

Comment:
file was previously called `ifFieldRequested`. I made it accept dot notation for nested fields (not foolproof - like if you query by `node`!) so that I could use it in the artworksConnection resolver.